src/enumerable.cr

```diff
@@ -1384,6 +1384,22 @@ module Enumerable(T)
     result
   end
 
+  # Returns a `Hash` with counts the elements in the collection.
```

Comment:
What do you think about using the same docs as Ruby:

https://github.com/ruby/ruby/blob/trunk/enum.c#L1004-L1013

Reply:
Sounds good 👍 Applied in https://github.com/crystal-lang/crystal/pull/7921/commits/49d9c6880ce0578eea3b17249130b239f5acf660